src/Bitcoin/Util.hs

```diff
@@ -160,7 +149,7 @@ maybeToEither err = maybe (Left err) Right
 
 -- | Lift a 'Maybe' computation into the 'ExceptT' monad.
 liftMaybe :: Monad m => b -> Maybe a -> ExceptT b m a
-liftMaybe err = liftEither . maybeToEither err
+liftMaybe err = ExceptT . pure . maybeToEither err
```

Comment:
nit: This is a good use for `except`

Reply:
been coding haskell since 2017 and only today learned that `except` exists

Reply:
Hey I'm also a 2017 vintage Haskeller!